CoEpi/common/DateFormatters.swift

```diff
@@ -0,0 +1,17 @@
+import UIKit
+
+class DateFormatters {
+    static let dateHoursMins: DateHoursMinsFormatter = DateHoursMinsFormatter()
+}
+
+final class DateHoursMinsFormatter: DateFormatter {
+
+    override init() {
+        super.init()
+        dateFormat = "dd.MM.yyyy HH:mm"
```

Comment:
This is only "correct" in Europe. We should use the locale date format instead, or something universal like yyyy-MM-dd HH:mm TZ.